--- net/minecraft/server/MinecraftServer.java
+++ net/minecraft/server/MinecraftServer.java
@@ -96,7 +96,7 @@
 		var1.setDaemon(true);
 		var1.start();
 		ConsoleLogManager.init();
-		logger.info("Starting minecraft server version 1.2.5");
+		logger.info("Starting minecraft server version Unarchived");
 		if(Runtime.getRuntime().maxMemory() / 1024L / 1024L < 512L) {
 			logger.warning("To start the server with more ram, launch it as \"java -Xmx1024M -Xms1024M -jar minecraft_server.jar\"");
 		}
@@ -527,7 +527,7 @@
 	}
 
 	public String getVersionString() {
-		return "1.2.5";
+		return "Unarchived";
 	}
 
 	public int playersOnline() {
